rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -165,7 +165,7 @@ object SystemProcesses {
             params                                                  <- runtimeParametersRef.get
             DeployParameters(codeHash, phloRate, userId, timestamp) = params
             _ <- produce(
-              Seq(codeHash, phloRate, userId, timestamp),
+                  Seq(codeHash, phloRate, userId, timestamp),
```

Comment:
We should do something about automatic formatting! I know it's not your fault but I'm just pointing at the issue